acceptance/tests/resource/package/ips/should_be_updateable_and_unholdable_at_same_time.rb

```diff
@@ -0,0 +1,41 @@
+test_name "Package:IPS test for updatable holded package"
+confine :to, :platform => 'solaris-11'
+
+tag 'audit:high',
+    'audit:refactor',  # Use block style `test_name`
```

Comment:
You can remove `'audit:refactor'` and use `test_name...do` and `step...do` blocks